test/round-tracker.test.js

```diff
@@ -1,23 +1,27 @@
 import assert from 'node:assert'
 import pg from 'pg'
-import { TASKS_PER_ROUND, getRoundStartEpoch, mapCurrentMeridianRoundToSparkRound } from '../lib/round-tracker.js'
+import { TASKS_PER_ROUND, getRoundStartEpoch, mapCurrentMeridianRoundToSparkRound, startRoundTracker } from '../lib/round-tracker.js'
 import { migrate } from '../lib/migrate.js'
 import { assertApproximately } from './test-helpers.js'
 import { createMeridianContract } from '../lib/ie-contract.js'
 
 const { DATABASE_URL } = process.env
 
 describe('Round Tracker', () => {
+  /** @type {pg.Pool} */
+  let pgPool
+  /** @type {pg.PoolClient} */
   let pgClient
 
   before(async () => {
-    pgClient = new pg.Client({ connectionString: DATABASE_URL })
-    await pgClient.connect()
+    pgPool = new pg.Pool({ connectionString: DATABASE_URL })
+    pgClient = await pgPool.connect()
```

Comment:
Existing tests use a single connection, I am not sure if they require that.

The new test needs `pgPool`, or at least that's what the type says.

To keep the changes minimal, I am changing the test setup to offer both `pgPool` and `pgClient`.